.packit.yaml

```diff
@@ -41,7 +41,8 @@ jobs:
     epel-7-x86_64:
       distros: [centos-7, oraclelinux-7]
     epel-8-x86_64:
-      distros: [centos-8.4, centos-8-latest, oraclelinux-8.4, oraclelinux-8.6]
+      # TODO add oraclelinux-8.6 when the 8.7 is officially out
```

Comment:
```suggestion
      # TODO add oraclelinux-8.7 when the 8.7 is officially out
```
hehe

Reply:
This is just a temporary #TODO before we get the 'official' 8.7 image to the testing farm.
For the time being, we will use the 8.6 compose, and update that using the ansible role.
When the 8.7 is available on TF, we will then update the `distro` value to the `oraclelinux-8.7`
I guess the comment is confusing.

Reply:
A bit! If that's the case, we can leave the 8.6 enabled and just change later to 8.7 as you mentioned.

Reply:
A will try to change the comment to make it a bit clearer